_includes/manuals/3.14/1.2_release_notes.md

```diff
@@ -21,10 +21,139 @@ For more details, see the [removal RFC](https://community.theforeman.org/t/drop-
+* vmware Create controller select freezes the page  ([#38209](https://projects.theforeman.org/issues/38209))
+* form_select_f auto selects first option ([#38183](https://projects.theforeman.org/issues/38183))
+* Hide search submit button when not submittable   ([#38141](https://projects.theforeman.org/issues/38141))
+
+#### Installer
+* Stop managing postgresql-evr extension ([#37680](https://projects.theforeman.org/issues/37680))
+* New PXE loader "Grub2 UEFI SecureBoot (target OS)" ([#36940](https://projects.theforeman.org/issues/36940))
+
+#### Installer - Foreman modules
+* use lowercase FQDN in SSL CN comparison for pulpcore auth ([#38110](https://projects.theforeman.org/issues/38110))
+* On large deployments puma auto tuning results in too many workers for PostgreSQL connections ([#38085](https://projects.theforeman.org/issues/38085))
+
+#### Installer - foreman-installer script
+* Java detection doesn't handle Java 11 and newer and always stops the service ([#38012](https://projects.theforeman.org/issues/38012))
+* Halt installer run if the evr extension in remote DBs has the wrong permissions before upgrade ([#37883](https://projects.theforeman.org/issues/37883))
+* Change evr extension ownership to foreman via installer ([#37717](https://projects.theforeman.org/issues/37717))
+
+#### Packaging - RPMs
+* Drop EL7 client support ([#38034](https://projects.theforeman.org/issues/38034))
+
+#### SELinux
+* allow smart-proxy with PuppetCA to read some etc files ([#37999](https://projects.theforeman.org/issues/37999))
+* Remove dependency on unconfined selinux module ([#37968](https://projects.theforeman.org/issues/37968))
+
+#### Smart Proxy
+* New PXE loader "Grub2 UEFI SecureBoot (target OS)" ([#36833](https://projects.theforeman.org/issues/36833))
+
+#### Smart Proxy - Realm
+* rexml is not a default gem on ruby 3 anymore ([#38157](https://projects.theforeman.org/issues/38157))
+
+*A full list of changes in 3.14.0 is available via [Redmine](https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f[]=cf_12&op[cf_12]=%3D&v[cf_12]=1907)*
+
 ### Contributors
 
 We'd like to thank the following people who contributed to the Foreman {{page.version}} release:
 
-<!-- update scripts/committers.rb with the correct versions and dates and fill this in -->
+Adam Lazik,Adam Růžička,Aneta Šteflová Petrová,Archana Kumari,Bastian Schmidt,Ben Erickson,Bernhard Suttner,Brenden Wood,Chris Roberts,Cole Higgins,Eric Helms,Evgeni Golov,Ewoud Kohl van Wijngaarden,Francesco Di Nucci,Gaurav Talreja,Girija Soni,Hao Yu,Ian Ballou,Jan Löser,Jeremy Lenz,Leos Stejskal,Lucy Fu,Maria Agaphontzev,Markus Bucher,Markus Reisner,Martin Alfke,Martin Spiessl,Matthew Davis,Maximilian Kolb,Nadja Heitmann,Nofar Alfassi,Oleh Fedorenko,Partha Aji,Patrick Creech,PopiBrossard,Samir Jha,Sayan Das,Shimon Shtein,Takashi Kajinami,Tim Meusel,VHS,akumari,dosas,girijaasoni
```

Comment:
Somehow the spaces are all missing. I'd also prefer to maintain it multi line, just like we do in foreman-documentation. If needed, please also make it easier to do so in the future by changing the script.